cmd/spire-server/cli/federation/update.go

```diff
@@ -56,10 +60,24 @@ func (c *updateCommand) Run(ctx context.Context, env *common_cli.Env, serverClie
 		return fmt.Errorf("request failed: %w", err)
 	}
 
+	return c.printer.PrintProto(resp)
+}
+
+func (c *updateCommand) prettyPrintUpdate(env *commoncli.Env, results ...interface{}) error {
+	federationRelationships, err := getRelationships(c.config, c.path)
+	if err != nil {
+		return err
+	}
```

Comment:
relationships are getted twice?

Reply:
ops, I just let it pass, we need these relationships for building the request and them for composing the result in `pretty` format, im going to update it to remove redundant calls

Reply:
d2a3aab